### PR TITLE
Merge master i3s

### PR DESCRIPTION
**Pull request by tmiotto**
### Description
Merges `master_i3s` into `master` since oneview-sdk-ruby v4.1.0 now provides stable Image Streamer support.

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,24 +23,38 @@ Then use any of the resources provided by this cookbook.
...
+
+1. Set the resource property `api_version`. See the [Resource Properties](#resource-properties) section below.
+2. Set the client parameter `api_version`. If this parameter is set, it will select the required API version based on the client. Notice if you choose to pass the client as an OneviewSDK object, it will have, by default, the api_version set, even if you do not directly specify it.
+3. If none of the previous alternatives are set, it defaults to the `node['oneview']['api_version']`. See the [Atributes](#attributes).
+
+Be aware of the precedence of these methods! The higher priority goes to setting the resource property, followed by the client parameter, and at last the node value as the default, i.e. *Property > Client > Attribute*. (e.g. If you set the resource property `api_version` to *200*, and set the client parameter `api_version` to *300*, it will use the module **API200**, since the resource property takes precedence over the client parameter)
 
 ## Attributes
 
  - `node['oneview']['ruby_sdk_version']` - Set which version of the SDK to install and use. Defaults to `'~> 4.0'`
```
Comment (jsmartt) [inline README.md]: @tmiotto , do we need to update this to `~> 4.1`? (And in the attributes file)

**Comment by jsmartt** [comment]
Also, are there any other breaking changes that we should make before doing this? I don't want to have to do another major release for a while if we can avoid it.

File: README.md (hunk 1 above)
Comment (tmiotto) [reply README.md]: We don't need but it's better to do it.

**Comment by tmiotto** [comment]
Actually we got one right now...

> [2017-02-24T18:54:04+00:00] WARN: Property `state` of resource `oneview_enclosure` overwrites an existing method. Please use a different property name. This will raise an exception in Chef 13. (CHEF-11)/tmp/d20170224-16231-1pmns0b/cookbooks/oneview/resources/enclosure.rb:16:in `class_from_file'.

**Comment by tmiotto** [comment]
I don't know why, but one unit test started failing...

**Comment by jsmartt** [comment]
I got that too when I was running it against 4.1.0. I assume something changed in that version

**Comment by tmiotto** [comment]
Got it. The I3S_TOKEN was removed.

**Comment by jsmartt** [comment]
Cool. So do we wan to fix that `state` property while we're at it too?

**Comment by tmiotto** [comment]
Since we are stacking the breaking changes at this version, the refresh had to be refactored on managed_san also.

**Review by jsmartt — APPROVED** [review]
Looks good 👍 Are we ready to release @tmiotto ?

**Comment by tmiotto** [comment]
Oh yeah! :shipit:

**Pull request merged**